showcase/app/templates/components/rich-tooltip.hbs

```diff
@@ -0,0 +1,1095 @@
+              </RT.Bubble>
+            </Hds::RichTooltip>
+            sit amet.
+          </Shw::Outliner>
+        </SG.Item>
+      {{/each}}
+    {{/let}}
+  </Shw::Grid>
+
+  <Shw::Divider />
+
+  <Shw::Text::H2>Interaction states</Shw::Text::H2>
+
+  <Shw::Text::H3>Text + Icon</Shw::Text::H3>
+
+  <Shw::Text::H4>Block</Shw::Text::H4>
+
+  <Shw::Grid @columns={{4}} as |SG|>
+    {{#each @model.STATES as |state|}}
+      <SG.Item @label="{{capitalize state}}" class="shw-component-link-inline-state-samples">
+        <Hds::RichTooltip as |RT|>
+          <RT.Toggle
+            @text="Lorem ipsum dolor"
+            @isInline={{false}}
+            @icon="info"
+            @size="medium"
+            mock-state-value={{state}}
+          />
+          <RT.Bubble>
+            <Shw::Placeholder @text="generic content" @height="30" />
+          </RT.Bubble>
+        </Hds::RichTooltip>
+      </SG.Item>
+    {{/each}}
+  </Shw::Grid>
+
+  <Shw::Text::H4>Inline</Shw::Text::H4>
+
+  <Shw::Grid @columns={{4}} as |SG|>
+    {{#each @model.STATES as |state|}}
+      <SG.Item @label="{{capitalize state}}" class="shw-component-link-inline-state-samples">
+        <Hds::Text::Body @size="200" @tag="p" {{style margin-bottom="16px"}}>
+          Lorem
+          <Hds::RichTooltip as |RT|>
+            <RT.Toggle @text="ipsum dolor" @isInline={{true}} @icon="info" @size="medium" mock-state-value={{state}} />
+            <RT.Bubble>
+              <Shw::Placeholder @text="generic content" @height="30" />
+            </RT.Bubble>
+          </Hds::RichTooltip>
+          sit amet
+        </Hds::Text::Body>
+        <Hds::Text::Body @size="200" @tag="p" {{style margin-bottom="16px"}}>
+          Lorem ipsum dolor sit
+          <Hds::RichTooltip as |RT|>
+            <RT.Toggle
+              @text="amet consectetur"
+              @isInline={{true}}
+              @icon="info"
+              @size="medium"
+              mock-state-value={{state}}
+            />
+            <RT.Bubble>
+              <Shw::Placeholder @text="generic content" @height="30" />
+            </RT.Bubble>
+          </Hds::RichTooltip>
+          adipisicing elit
+        </Hds::Text::Body>
+        <Hds::Text::Body @size="200" @tag="p">
+          Lorem ipsum
+          <Hds::RichTooltip as |RT|>
+            <RT.Toggle
+              @text="dolor sit amet consectetur adipisicing elit"
+              @isInline={{true}}
+              @icon="info"
+              @size="medium"
+              mock-state-value={{state}}
+            />
+            <RT.Bubble>
+              <Shw::Placeholder @text="generic content" @height="30" />
+            </RT.Bubble>
+          </Hds::RichTooltip>
+        </Hds::Text::Body>
+      </SG.Item>
+    {{/each}}
+  </Shw::Grid>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>Generic</Shw::Text::H3>
+
+  <Shw::Text::Body>
+    In this case the "toggle" doesn't have special styles of its own other than for the "focus" state.
```

Comment:
```suggestion
    Not for use in production, but potentially for future use cases. Note that In this case the "toggle" doesn't have special styles of its own other than for the "focus" state.
```

Suggesting some additional wording that makes it clear that we don't intend consumers to use this and it's for future use cases that might come up and need extra styling.

Reply:
added